### DISCUSSION TITLE
Batch audit of handshakes

### DISCUSSION THREAD

**Discussion by lavinkabul** [aircrack-ng/aircrack-ng#2504] [General]
I already have 35 access points in my cap file. How to make a sequential iteration of all points without choosing separately? That is, run it in the evening, and the next day see how successful the password audit of all access points is

**Comment by Mister-X-** [comment]
There are multiple ways to do this. Here is a couple I can think of:

- You can extract the handshakes of each AP, put them in a pcap, and have aircrack-ng run them.
- Script it. Parse the PCAP to extract each BSSID, and run aircrack-ng on each of them.
